pom.xml

```diff
@@ -78,37 +84,6 @@
         </plugins>
     </build>
 
-    <reporting>
-        <plugins>
-            <plugin>
-                <artifactId>maven-javadoc-plugin</artifactId>
-                <configuration>
-                    <links>
-                        <link>https://hudson.dev.java.net/nonav/javadoc/</link>
-                        <link>https://stapler.dev.java.net/nonav/javadoc/</link>
-                    </links>
-                </configuration>
-            </plugin>
-
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
```

Comment:
Default Parent POM does not offer Cobertura (it offers jacoco instead). But I doubt anybody really cares about the code coverage in the current state